prime-router/src/main/kotlin/Report.kt

```diff
@@ -76,26 +76,16 @@ data class ReportStreamFilterResult(
     val filterArgs: List<String>,
     val filteredTrackingElement: String,
     val filteredIndex: Int,
-    val filterType: ReportStreamFilterType?,
-    override val type: ActionLogDetailType = ActionLogDetailType.TRANSLATION
+    val filterType: ReportStreamFilterType?
 ) : ActionLogDetail {
+    override val scope = ActionLogScope.translation
     companion object {
         // Use this value in logs and user-facing messages if the trackingElement is missing.
-        val DEFAULT_TRACKING_VALUE = "MissingID"
+        const val DEFAULT_TRACKING_VALUE = "MissingID"
     }
 
-    override fun toString(): String {
-        return "For $receiverName, filter $filterName$filterArgs" +
-            " filtered out item $filteredTrackingElement at index $filteredIndex"
-    }
-
-    override fun detailMsg(): String {
-        return toString()
-    }
-
-    override fun groupingId(): String {
-        return receiverName
-    }
+    override val message = "For $receiverName, filter $filterName$filterArgs" +
+        " filtered out item $filteredTrackingElement at index $filteredIndex"
```

Comment:
From @jimduff-usds - Remove at index as the index numbering is incorrect when seen from a filter.  Do this for the reports response too.

Reply:
Done